docs/modules/ROOT/pages/references/operators.adoc

```diff
@@ -0,0 +1,233 @@
+= Operators
+
+An OpenShift 4 cluster comes with many operators and controllers preinstalled.
+This page gives an overview of all installed operators on a new cluster and their purpose.
+
+Most of the operators are described to some extent in the https://docs.openshift.com/container-platform/4.4/operators/operator-reference.html[official documentation].
+
+[cols="2,2a,2a,1", stripes=hover]
+|===
+|Description |CRDs |Namespaces |Commodore
+
+4+|**Cluster**
+
+|The https://docs.openshift.com/container-platform/4.4/operators/operator-reference.html#cluster-version-operator_red-hat-operators[Cluster Version Operator] handles updates for a cluster.
+|* `ClusterVersion` - is the configuration for automatic updates of the cluster
+|* `openshift-cluster-version`
+|
+
+|The https://docs.openshift.com/container-platform/4.4/operators/operator-reference.html#dns-operator_red-hat-operators[DNS Operator] deploys and manages CoreDNS on a cluster.
+|* `DNS.config.openshift.io/v1` - holds cluster-wide information about DNS
+* `DNS.operator.openshift.io/v1` - manages the CoreDNS component
+|* `openshift-dns-operator`
+* `openshift-dns`
+|
+
+|The https://github.com/openshift/service-ca-operator[Service CA Operator] manages the serving cert signer, ConfigMap CA bundle injector and generic CA bundle injector.
+|* `ServiceCA` - provides information to configure an operator to manage the service cert controllers
+|* `openshift-service-ca-operator`
+* `openshift-service-ca`
+|
+
+|The https://docs.openshift.com/container-platform/4.4/operators/operator-reference.html#cloud-credential-operator_red-hat-operators[Cloud Credential Operator] manages cloud provider credentials on a cluster.
+|* `CredentialsRequest` - request cloud credentials
+|* `openshift-cloud-credential-operator`
+|
+
+
+4+|https://docs.openshift.com/container-platform/4.4/architecture/control-plane.html[**Control Plane**]
```

Comment:
Semantically it would be better to have the groups sections each holding a table on their own. I just tripped over this. It took me quite some time to figure, that this is a heading of some sort and not a regular table item.